activerecord/lib/active_record/core.rb

```diff
@@ -461,6 +337,30 @@ def find_by!(*args) # :nodoc:
         find_by(*args) || raise(RecordNotFound.new("Couldn't find #{name}", name))
       end
 
+      def default_timezone # :nodoc:
+        ActiveRecord.default_timezone
+      end
+
+      def maintain_test_schema # :nodoc:
+        ActiveRecord.maintain_test_schema
+      end
+
+      def reading_role # :nodoc:
+        ActiveSupport::Deprecation.warn(<<~MSG)
+          ActiveRecord::Base.reading_role is deprecated and will be removed in Rails 7.0.
```

Comment:
does this means this will be released in 6.2 or 6.1.x ?

Reply:
No it was a typo that got corrected later.